src/auth/auth.controller.ts

```
@@ -110,5 +108,13 @@ async requestPasswordReset(@Body('email') email: string) {
     return this.authService.resetPassword(body.token, body.newPassword);
   }
 
-
+  @Get('users')
+  @UseGuards(AuthGuard('jwt'), AdminGuard)
+  @ApiOperation({ summary: 'Get all users with pagination and filters (admin only)' })
```

ESLint

[error] Replace `·summary:·'Get·all·users·with·pagination·and·filters·(admin·only)'` with `⏎····summary:·'Get·all·users·with·pagination·and·filters·(admin·only)',⏎·`

(prettier/prettier)